easybuild/easyconfigs/p/PCRE/PCRE-8.42-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'PCRE'
+version = '8.42'
+
+homepage = 'http://www.pcre.org/'
+
+description = """
+ The PCRE library is a set of functions that implement regular expression
+ pattern matching using the same syntax and semantics as Perl 5.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/']
```

Comment:
@JackPerdue Where possible, we prefer `http://` over `ftp://`; in this case, we could include both, but have `http://` first:

```python
source_urls = [
    'https://ftp.pcre.org/pub/pcre/',
    'ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/',
]

Reply:
Fixed